### PR TITLE
Add ”Speakers Guide”

### LINKED ISSUE #75: Speakers Guide Page
Like this one, but with updated content:
http://2015.cssconf.eu/speakers-guide/

We never link to this page, we just send it to the speakers.

### PR DESCRIPTION
**Pull request by lukaszklis**
Fixes #75

### CODE REVIEW

File: speakers-guide.html [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    </section>
+
+    <section>
+      <div class="row">
+        <div class="col-xs-12 col-sm-8 col-sm-offset-2">
+          <h2 class="color--blue">
+            The Team: Here to Help!
+          </h2>
+          <p>Should you have any questions or requests, feel free to speak to us any time!</p>
+          {% include organizers.html %}
```
Comment (lukaszklis) [inline speakers-guide.html:31]: We might want to add some additional `margin-top` for the `.c-team-photos` (will require updating 2017-assets).

File: speakers-guide.html [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+<main>
+  <div class="container">
+    <section>
+      <div class="row">
+        <div class="col-xs-12 col-sm-8 col-sm-offset-2">
+          <p class="intro color--blue">
+            Dear CSSconf EU Speakers,
+          </p>
+          <p>
+            Thanks again for being part of CSSconf EU 2015, we're really looking forward to meeting you in Berlin soon! We created this page to answer any questions you may have, and generally to help make your travels and your stay here as smooth as possible. Please let us know if there's anything that hasn't been addressed here and feel free to contact us any time should you need help or assistance!
```
Comment (Kriesse) [inline speakers-guide.html]: 2015 > 2017

File: speakers-guide.html [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+      </div>
+    </section>
+
+    <section>
+      <div class="row">
+        <div class="col-xs-12 col-sm-8 col-sm-offset-2">
+          <h2 class="color--blue">
+            Your Hotel
+          </h2>
+          <p class="intro">Ibis am Ostbahnhof | An der Schillingbrücke 2 | 10243 Berlin</p>
```
Comment (Kriesse) [inline speakers-guide.html]: let's comment this out, we don't have that info yet (i'll request it!)

File: speakers-guide.html [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+          <p>
+            We have booked a room for you from Friday to Sunday (unless you confirmed other dates with us or have private accommodation). If your company pays for your accommodation: Great! Please let us know at <a href="mailto:contact@cssconf.eu">contact@cssconf.eu</a>, and we’ll issue an invoice.
```
Comment (Kriesse) [inline speakers-guide.html]: correct for 2017: from Thursday to Monday

File: speakers-guide.html [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+      <div class="row">
+        <div class="col-xs-12 col-sm-8 col-sm-offset-2">
+          <h2 class="color--blue">
+            Reimbursements
+          </h2>
+          <p>
+            We will reimburse you for your travel-related expenses. If your company has offered to pay: Awesome, thanks! We’d be happy to list them as a supporter on our website, so just let us know if that's the case.
+          </p>
+          <p>
+            To get reimbursed for your travel expenses, we ask that you email the following to <a href="mailto:contact@cssconf.eu">contact@cssconf.eu</a>:
```
Comment (Kriesse) [inline speakers-guide.html]: and also to travel@jsconf.eu

File: speakers-guide.html [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+                <li>
+                  Additional information such as the address of the bank is helpful too, to make sure the international wire goes through without issues.
+                </li>
+              </ul>
+            </li>
+            <li>
+              Please add the receipts for your flight(s), taxi, bus or other public transport rides to your invoice.
+            </li>
+            <li>
+              Please send the invoice and all receipts via email to <a href="mailto:contact@cssconf.eu">contact@cssconf.eu</a>. We will then forward them to our accountants and get the wire initiated.
```
Comment (Kriesse) [inline speakers-guide.html]: and also to travel@jsconf.eu

File: speakers-guide.html [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+          </h2>
+          <ul class="list-simple">
+            <li>
+              The day will start at 8:30am with breakfast and registration; talks will begin at 9:30am. It’s advisable that you arrive before 9:30am, so that we can set up your notebook for a test run and make sure everything works as it should. Here's the <a href="/schedule/">full schedule and line-up</a>.
+            </li>
+            <li>
+              As a speaker, you won’t have to queue at the venue for check-in, you can come to the registration desk directly. You’ll then get your speaker bag with your conference pass.
+            </li>
+            <li>
+              Projector resolution is full HD (1080p), that is 1920x1080.
```
Comment (Kriesse) [inline speakers-guide.html]: Those need to be fact-checked

File: speakers-guide.html [hunk 8]
```diff
@@ -0,0 +1,266 @@
...
+      </div>
+    </section>
+
+    <section>
+      <div class="row">
+        <div class="col-xs-12 col-sm-8 col-sm-offset-2">
+          <h2 class="color--blue">
+            Parties and Social Events
+          </h2>
+          <h3>Friday, 7pm: JSconf Dinner</h3>
```
Comment (Kriesse) [inline speakers-guide.html]: Those Dates need updating, they should be the same as on /info

**Review by Kriesse — CHANGES_REQUESTED** [review]
Awesome start! We can merge and publish this now and implement the copy changes together.

File: speakers-guide.html [hunk 9]
```diff
@@ -0,0 +1,256 @@
...
+          </p>
+          <h3>Monday, May 8: relax.js Brunch, 10am</h3>
+          <p>
+            TBD
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section>
```
Comment (lukaszklis) [inline speakers-guide.html:213]: Don’t know if we need this section at the moment. Shall I remove it for now?

**Comment by lukaszklis** [comment]
@Kriesse updated the PR. :)

**Comment by lukaszklis** [comment]
Will re-open from origin.

**Pull request closed without merging**